service/frontend/redirection_interceptor.go

```diff
@@ -42,9 +44,13 @@ import (
 	"go.temporal.io/server/common/rpc/interceptor"
 )
 
-var (
+const (
+	dcRedirectionContextHeaderName = "xdc-redirection"
```

Comment:
nit: Define this in common/headers.go

Reply:
this header will only be used by the redirection interceptor (not a common header)
do you still prefer to move this header definition to `common/headers.go`?

Reply:
ok. I was thinking define all headers in one place. I am fine with either way.